composeApp/src/commonMain/kotlin/designsystem/login/LoginButton.kt

```diff
@@ -0,0 +1,42 @@
+package designsystem.login
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import designsystem.colors.ButtonColor
+import dev.icerock.moko.resources.compose.stringResource
+import instaclone.resources.MR
+import login.horizontalPadding
+
+@Composable
+fun LoginButton(loginButtonState: MutableState<ButtonColor>, inputsState: MutableMap<String, Boolean>) {
```

Comment:
For the login button, I think we just need to receive the arguments: `onClick`, and `modifier`.

Reply:
- I'm a little confused here, are you saying  we need these two extra parameter on top of the ones are already or just those two? I'll modify as you meaning the latter, let me know if that's the wrong reasoning.
- I'll add the onClick() call, but I feel like that will make the LoginScreen more convoluted, or I might have misunderstood the request here